app/src/view/DatePicker.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Date picker, wraps ReactDatePicker for now
+ * @author scjody
+ * @since May 2020
+ * @flow
+ */
+
+import React from 'react';
+import { createPortal } from 'react-dom';
+import ReactDatePicker from 'react-datepicker';
+import { makeStyles } from '@material-ui/core/styles';
+import { formatISO, parseISO } from 'date-fns';
+
+import 'react-datepicker/dist/react-datepicker.css';
+
+type FormTarget = { target: { name: string, value: string }};
+type P = {
+  name: string,
+  onChange: FormTarget => void,
+  selected: string,
+  label: string,
+};
+
+const useStyles = makeStyles((theme) => {
+  const borderColor = theme.palette.type === 'light' ? 'rgba(0, 0, 0, 0.23)' : 'rgba(255, 255, 255, 0.23)';
```

Comment:
Is there no `theme.palette` property that holds this value? Manually specifying it based on type seems wrong.

Reply:
Unfortunately there isn't. I took this computation directly from [Material UI's OutlinedInput](https://github.com/mui-org/material-ui/blob/fa7f197db397572634e5be894a0971453867bdd4/packages/material-ui/src/OutlinedInput/OutlinedInput.js#L11) because anything else would look different from the other inputs.

The computation is [repeated a few times](https://github.com/mui-org/material-ui/search?q=%22rgba%280%2C+0%2C+0%2C+0.23%29%22&unscoped_q=%22rgba%280%2C+0%2C+0%2C+0.23%29%22) throughout their code so I'm surprised they haven't pulled this out into the theme, but there you go.